docs/source/disable-warnings.py

```diff
@@ -0,0 +1,2 @@
+import warnings
+warnings.filterwarnings('ignore')
```

Comment:
Just thinking out loud... do we want to disable *all* warnings? Is it possible to disable just `DeprecationWarning`s? I know warnings in the past have been useful in showing where some things are not as we expected, could be out of scope for this PR though.

Reply:
Agree on the idea but I also think it's best suited for a different PR. We'll have to modify [deprecate_arg](https://github.com/alteryx/evalml/blob/main/evalml/utils/gen_utils.py#L440) to issue a `DeprecationWarning` for starters which, although is a small change, seems out of scope for this PR. I think the intent was a refactor without changing behavior.

Reply:
Great idea! Filed a new issue [here](https://github.com/alteryx/evalml/issues/2107)